index.ts

```diff
@@ -97,14 +97,15 @@ function processTree(sourceFile: ts.SourceFile, replacer:(node: ts.Node) => stri
 }
 
 export function generate(options: Options, sendMessage: (message: string) => void = function () {}) {
-	var baseDir = pathUtil.resolve(options.baseDir);
+	var baseDir = filenameToMid(pathUtil.resolve(options.baseDir));
 	var eol = options.eol || os.EOL;
 	var nonEmptyLineStart = new RegExp(eol + '(?!' + eol + '|$)', 'g');
 	var indent = options.indent === undefined ? '\t' : options.indent;
 	var target = options.target || ts.ScriptTarget.Latest;
+	var module = options.module || ts.ModuleKind.CommonJS;
 	var compilerOptions: ts.CompilerOptions = {
 		declaration: true,
-		module: ts.ModuleKind.CommonJS,
+		module: module,
```

Comment:
The type of output module shouldn’t matter since we’re only using the declaration information. Is there some case where you are experiencing a failure?